src/System.Security.Permissions/src/System/Data/Common/DBDataPermission.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security;
+using System.Security.Permissions;
+
+namespace System.Data.Common
+{
+    [Serializable]
+    public abstract class DBDataPermission : CodeAccessPermission, IUnrestrictedPermission
+    {
+        [Obsolete]
+        protected DBDataPermission() { }
+        protected DBDataPermission(DBDataPermission dataPermission) { }
+        protected DBDataPermission(DBDataPermissionAttribute attribute) { }
+        protected DBDataPermission(PermissionState state) { }
+        [Obsolete]
+        protected DBDataPermission(PermissionState state, bool blankPassword) { }
+        public bool AllowBlankPassword { get; set; }
+        public virtual void Add(string connectionString, string restrictions, KeyRestrictionBehavior behavior) { }
+        protected void Clear() { }
+        protected virtual DBDataPermission CreateInstance() => null;
+        public override IPermission Copy() => this;
```

Comment:
Seems about half and half whether Copy returns `null` (or equivalently `default<T>`) vs `this`. I think I would have done null, since Copy is a lie. I don't care either way.